routes/students.js

```diff
@@ -34,4 +34,8 @@ router.get('/kott', (req, res, next) => {
     res.render('students/kott', null)
 })
 
+router.get('/kott', (req, res, next) => {
```

Comment:
The route should be unique for every user and match what's written in `res.render`